rest-app/build.gradle

```diff
@@ -0,0 +1,5 @@
+apply plugin: 'java'
+dependencies {
+  compile project(':service')
+    compile group: 'org.springframework.boot', name: 'spring-boot-starter', version:'2.2.0.RELEASE'
```

Comment:
1. version shouldn't been declared here.
2. this dependency already assigned on a top level for all of subprojects